src/Uno.UWP/Storage/StorageFile.skia.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Windows.Storage
+{
+	partial class StorageFile
+	{
+		private static async Task<StorageFile> GetFileFromApplicationUri(CancellationToken ct, Uri uri)
+		{
+			if (uri.Scheme != "ms-appx")
+			{
+				throw new InvalidOperationException("Uri is not using the ms-appx scheme");
+			}
+
+			var path = uri.PathAndQuery.TrimStart(new char[] { '/' });
+
+			var baseDir = global::System.IO.Path.GetDirectoryName(global::System.Reflection.Assembly.GetExecutingAssembly().Location);
```

Comment:
Instead, use:

https://github.com/unoplatform/uno/blob/c4eaf68fee3502e5847446251412787ccea67ce7/src/Uno.UWP/ApplicationModel/Package.cs#L26

This code may need to change because of single-file packaging, and it's best to have it only at one location.

Reply:
Updated.